### PR TITLE
update api docs with more cimi info

### PR DESCRIPTION
**Pull request by loomis**
Connected to #34.

### CODE REVIEW

File: source/index.md.erb [hunk 1]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
...
+The resources under `/api` namespace do not support basic authentication and require an authentication cookie.
 
-Get instance names of a node in Run.
+<aside class="notice">
+From
+<a target="_blank" href="http://dmtf.org/sites/default/files/standards/documents/DSP0263_1.0.1.pdf">
+CIMI Model and RESTful HTTP-based Protocol documentation 5.17.13</a>
+<br />
+<br />
+A resource that represents the occurrence of an event within the managed infrastructure.
```
Comment (elegoff) [inline source/index.md.erb]: This sentence is unclear to me, should "that" be removed ?

File: source/index.md.erb [hunk 2]
```diff
@@ -15,19 +15,92 @@ search: true
...
+Services](http://shop.oreilly.com/product/9780596529260.do), and
+reuses the standard and universally supported HTTP protocol.
+
+SlipStream predates the standardization efforts for APIs and libraries
+within the cloud sector.  Consequently, the original API is unique to
+SlipStream. However, as technologies have changed and the standards
+have emerged, the SlipStream developers have decided to incrementally
+adopt the [Cloud Infrastructure Management
+Interface](https://www.dmtf.org/sites/default/files/standards/documents/DSP0263_2.0.0.pdf)
+(CIMI) maintained by DMTF. As a result, the full SlipStream API
```
Comment (loomis) [inline source/index.md.erb]: Done.

File: source/index.md.erb [hunk 3]
```diff
@@ -15,19 +15,92 @@ search: true
...
+patterns of the legacy API.**
+
+Action | HTTP Method | URL
+------ | ----------- | ---
+Search | GET | resource collection
+Create | POST | resource collection
+Read | GET | resource
+Update | POST | resource
+Delete | DELETE | resource
+
```
Comment (loomis) [inline source/index.md.erb]: Indeed PUT is used for Update.  Changed in the text.

File: source/index.md.erb [hunk 4]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
 curl https://nuv.la -H "Content-Type: application/xml"
```
Comment (loomis) [inline source/index.md.erb]: Added /run as an example.

File: source/index.md.erb [hunk 5]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
...
 
-SlipStream's REST design was inspired from Richardson and Ruby's excellent book:
-[RESTful Web Services](http://shop.oreilly.com/product/9780596529260.do). They present the
-Resource Oriented Architecture (ROA) concept, which guided several design choices, including
-the resources-based architecture underpinning SlipStream.  For example, each version of an image,
-deployment and project can be uniquely identified via a unique URI.
+## Skeleton (*new*) Resources
 
+```shell
+curl https://nuv.la/module/examples/new?category=Image login -d username=<USER> -d password=<PASSWORD>
```
Comment (loomis) [inline source/index.md.erb]: Will create a ticket for this.

File: source/index.md.erb [hunk 6]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
...
+# List all of the credential creation mechanisms
+#
+curl https://nuv.la/api/credential-template
+```
 
 ```shell
-curl https://nuv.la/module/<module>/ -u <user>:<password>
+# You can get more details by using the "describe" action.
+#
+curl https://nuv.la/api/credential-template/generate-api-key/describe
```
Comment (loomis) [inline source/index.md.erb]: Added note to the examples.

File: source/index.md.erb [hunk 7]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
...
+"dsa"), and the fingerprint of the key itself.
+
+These resources can be created either by providing the public key of
+an existing SSH key pair or by having the server generate a new SSH
+key pair.  In the second case, the server will provide the private key
+in the 201 (created) response.
 
-`GET https://nuv.la/run/<run-uuid>`
+<aside class="notice">
+When the server generates a new SSH key pair, the server returns the
```
Comment (loomis) [inline source/index.md.erb]: You can create any number of SSH public key credentials on the server.  I've added a sentence about this.

File: source/index.md.erb [hunk 8]
```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
...
+                         }
+}
+```
 
 ```shell
-curl https://nuv.la/run -d refqname=module/examples/an-image -d type=Run -u <user>:<password> -X POST -H "Content-Type: text/plain" -D -
+# Be sure to get the URL from the cloud entry point!
+# The cookie options allow for automatic management of the
+# SlipStream authentication token (cookie).
+curl https://nuv.la/api/credential \
```
Comment (loomis) [inline source/index.md.erb]: Fixed now.

File: source/index.md.erb (hunk 1 above)
Comment (loomis) [inline source/index.md.erb]: Entirely remove this aside and provided a (hopefully) clearer description of the Event resource.

File: source/index.md.erb (hunk 5 above)
Comment (loomis) [inline source/index.md.erb]: #36

**Comment by loomis** [comment]
@elegoff Thanks for the review.

**Comment by loomis** [comment]
PR merged and published.

**Pull request merged**